### ISSUE TITLE
Line number mismatches

### ISSUE THREAD

**Issue by danni** [koterpillar/aloe#35]
Tracebacks occasionally have the wrong line numbers for features and steps. Precise conditions to reproduce this are not yet determined.

**Comment by koterpillar** [comment]
Are you using outlines? https://github.com/koterpillar/aloe/blob/master/aloe/testclass.py#L215

**Comment by koterpillar** [comment]
Outlines will be fixed after #40.

**Comment by koterpillar** [comment]
Another issue of this in #83. Closing as there's no useful information here, reopen/open a new one if there's a test case.
